Modules/_importlib.c

```diff
@@ -0,0 +1,39 @@
+#define PY_SSIZE_T_CLEAN
+#include "Python.h"
+#include "clinic/_importlib.c.h"
+
+static PyObject *
+is_lazy_import_impl(PyObject *module, PyObject *dict, PyObject *key)
+{
+    int res = PyDict_IsLazyImport(dict, key);
+
+    if (res == -1) {
+        PyErr_SetObject(PyExc_KeyError, key);
+        return NULL;
+    }
+    else if (res == 1) {
+        Py_RETURN_TRUE;
+    }
+    else {
+        Py_RETURN_FALSE;
+    }
```

Comment:
nit: no need for else's after return
```suggestion
    if (res == 1) {
        Py_RETURN_TRUE;
    }
    Py_RETURN_FALSE;
```